index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index2.css" />
+  </head>
+  <body>
+    <section class="first-section">
+      <div class="first-container">
+        <nav class="first-nav">
+          <h1 class="my-logo">My Logo</h1>
+          <div class="hamburger">
+            <img src="./images/Union.svg" alt="union background" />
+          </div>
+        </nav>
+
+        <h2 class="first-para">
+          I’m Mirela<br />
+          Glad to see <br />
+          you!
+        </h2>
```

Comment:
- [ ] [RQUIRED] I think it would be better if you use an `<h1>` element instead of the `<h2>` element ♻️ to emphasize that this is the most important heading in the page 🌟 For better Search Engines Optimisiation 🌐.